### PR TITLE
Add setContentlength() method to Phalcon\Http\Response

### PR DESCRIPTION
**Pull request by zamronypj**

### CODE REVIEW

File: phalcon/http/response.zep [hunk 1]
```diff
@@ -14,6 +14,7 @@
  +------------------------------------------------------------------------+
  | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
  |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ |          Zamrony P. Juhara <zamronypj@yahoo.com>                       |
```
Comment (sergeyklay) [inline phalcon/http/response.zep]: new author just for doing a patch?

File: phalcon/http/responseinterface.zep [hunk 2]
```diff
@@ -14,6 +14,7 @@
  +------------------------------------------------------------------------+
  | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
  |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ |          Zamrony P. Juhara <zamronypj@yahoo.com>                       |
```
Comment (sergeyklay) [inline phalcon/http/responseinterface.zep]: The same

File: phalcon/http/responseinterface.zep [hunk 3]
```diff
@@ -77,6 +78,15 @@ interface ResponseInterface
 	public function setContentType(string contentType, charset = null) -> <ResponseInterface>;
 
 	/**
+	 * Sets the response content-length
+	 *
+	 * @param int contentLength
+	 * @param string charset
+	 * @return \Phalcon\Http\ResponseInterface
```
Comment (sergeyklay) [inline phalcon/http/responseinterface.zep]: 1. `@param` twice
2. `@param`, `@retuurn` not needed here

File: phalcon/http/response.zep [hunk 4]
```diff
@@ -423,6 +424,23 @@ class Response implements ResponseInterface, InjectionAwareInterface
...
 
 	/**
+	 * Sets the response content-length
+	 *
+	 *<code>
+	 *	$response->setContentLength(2048);
+	 *</code>
+	 *
+	 * @param int contentLength
+	 * @return \Phalcon\Http\Response
```
Comment (sergeyklay) [inline phalcon/http/response.zep]: 1. `@param`, `@return` not needed here

File: phalcon/http/response.zep (hunk 1 above)
Comment (zamronypj) [inline phalcon/http/response.zep]: Why? I thought every contributors have right to put their name. I thought we value every contribution even smallest one? I do not get financial reward to do this work so putting my name is my reward. Your statement discourages people to contribute.

File: phalcon/http/responseinterface.zep (hunk 3 above)
Comment (zamronypj) [inline phalcon/http/responseinterface.zep]: 1. Yes @param string charset should not be there.
2. What do you mean?

File: phalcon/http/response.zep (hunk 4 above)
Comment (sergeyklay) [inline phalcon/http/response.zep]: We automatically generate the documentation based on the code. Zephir already knows how to do PHPDoc.

**Comment by sergeyklay** [comment]
@zamronypj Could you please update `CHANGELOG.md`

**Comment by sergeyklay** [comment]
@zamronypj Thank you for contributing. Few things..

1. Don't add changes to the `CHANGELOG.md` which do not change the API (eg `Remove unused  comment`)
2. Always add changes to the bottom of list
3. Could you please squash it into a single commit to keep the history clean?

I apologize for the inconvenience. Let me know if you need instructions.

File: phalcon/http/response.zep (hunk 1 above)
Comment (Jurigag) [inline phalcon/http/response.zep]: Basically andres and eduar are added as authors always beacause they are authors of frameworks and you can you as author if you are **creating** new file, but othey than that authors should stay the same.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**